cmd/support_archive/eec_fs_logs.go

```diff
@@ -0,0 +1,231 @@
+package support_archive
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/Dynatrace/dynatrace-operator/cmd/remote_command"
+	"github.com/Dynatrace/dynatrace-operator/pkg/logd"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/installconfig"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/labels"
+	"github.com/pkg/errors"
+	"golang.org/x/net/context"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	apilabels "k8s.io/apimachinery/pkg/labels"
+	clientgocorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+)
+
+const (
+	diagLogCollectorName = "fsLogCollector"
+	eecExtensionsPath    = "/var/lib/dynatrace/remotepluginmodule/log/extensions"
+	fileNotFoundMarker   = "<NOT FOUND>"
+	LabelEecPodName      = "dynatrace-extensions-controller"
+	eecContainerName     = "extensions-controller"
+)
+
+type fsLogCollector struct {
+	ctx                   context.Context
+	pods                  clientgocorev1.PodInterface
+	remoteCommandExecutor remote_command.Executor
+	config                *rest.Config
+	collectorCommon
+	appName            string
+	collectManagedLogs bool
+}
+
+var (
+	eecPodNotFoundError = errors.New("eec pod not found")
+)
+
+func newFsLogCollector(context context.Context, config *rest.Config, command remote_command.Executor, log logd.Logger, supportArchive archiver, pods clientgocorev1.PodInterface, appName string, collectManagedLogs bool) collector { //nolint:revive
+	return fsLogCollector{
+		collectorCommon: collectorCommon{
+			log:            log,
+			supportArchive: supportArchive,
+		},
+		ctx:                   context,
+		config:                config,
+		pods:                  pods,
+		appName:               appName,
+		collectManagedLogs:    collectManagedLogs,
+		remoteCommandExecutor: command,
+	}
+}
+
+func (collector fsLogCollector) Name() string {
```

Comment:
```suggestion
func (fc fsLogCollector) Name() string {
```

Reply:
name of the `logCollector` receivers changed from `collector` to `lc`
name of the `fsLogCollector` receivers changed from `collector` to `flc`